website/docs/r/elkaliases_index_aliases.markdown

```diff
@@ -8,7 +8,10 @@ description: |-
 
 # alkaliases\_index
 
-The ``elkaliases_index_aliases`` resource creates aliases in an index
+The ``elkaliases_index_aliases`` resource creates aliases in an existing index.
+
+
+~> **Note**: The index needs to already exist at deployment time. This means that you cannot create an index template and apply aliases to it directly. You need to create an index or data stream first.
```

Comment:
Can you add a note: This can be solved by declaring explicit dependency between the two resources?

Reply:
This is the point, it cannot be solved like this because it needs a manual operation between the two deployments.
Since an index template don't create a index and the aliases resource need an already existing template.